components/ai_chat/resources/page/stories/locale.ts

```diff
@@ -16,5 +16,6 @@ provideStrings({
   aboutDescription: 'The Brave AI Chat feature summarizes the content of a webpage you browse and includes a chat interface that allows you to make follow-up inquiries of the summarized content for that page.',
   aboutDescription_2: 'The summarized page and any questions you ask of the Chat feature are sent to the servers of an external API partner (no identifiers are sent with that query).',
   aboutNote: 'Please ensure that the page content and any follow-up inquiries do not contain personal or sensitive information. The accuracy of summaries and inquiry responses is not guaranteed and so you should not rely on any text related to health, personal safety or financial matters.',
-  acceptButtonLabel: 'Accept and begin'
+  acceptButtonLabel: 'Accept and begin',
+  pageContentWarning: 'Page content will be sent to Brave Leo along with your messages. Click the disconnect button on the right to start a new conversation about a different topic'
```

Comment:
I don't think we can use "on the right" since we have different UI directions for different locales. I would suggest simply removing that part.

Reply:
Removed